integration-tests/src/test/java/com/datastax/oss/driver/core/cql/PreparedStatementIT.java

```diff
@@ -474,6 +475,34 @@ public void should_infer_routing_information_when_partition_key_is_bound() {
         "DELETE FROM prepared_statement_test WHERE a = ?");
   }
 
+  @Test
+  @CassandraSkip // Functionality only available in Scylla
```

Comment:
You said that you saw that `@CassandraSkip` was not honored. I saw something like this when I fixing original integration tests and I worked around it this way: https://github.com/scylladb/java-driver/blob/7381fb9f8c5a27bee994194e686b60178977940e/integration-tests/src/test/java/com/datastax/oss/driver/core/cql/reactive/DefaultReactiveResultSetIT.java#L180

The original DataStax tests also did it that way, so I didn't spend any time debugging this, but thinking more about this I think you should try to debug this problem. What's strange is there are other annotations `@ScyllaSkip` annotations in this file and they are correctly honored.

Reply:
I found the issue. The adnotation is honored in `PreparedStatementIT`, but not in `LWTLoadBalancingIT` - in the latter it is only checked at class-level, not method-level. The difference seems to be `@Category(ParallelizableTests.class)`. Unfortunately, my new test class can't use this adnotation - for the test to be parallelizable it must use exactly 1 node, my tests have to use more. So I'll use the ` assumeThat(CcmBridge.SCYLLA_ENABLEMENT).isFalse();` solution (or rather: `assumeTrue(CcmBridge.SCYLLA_ENABLEMENT);`).